app/hooks/useShareCredentials.ts

```diff
@@ -9,7 +9,11 @@ export function useShareCredentials(): (credentials: CredentialRecordRaw[]) => v
   const { rawDidRecords } = useSelector<RootState, DidState>(({ did }) => did);
 
   if (rawDidRecords.length === 0) {
-    throw new Error('No DID generated. Something went wrong in wallet initialization.');
+    return (credentials: CredentialRecordRaw[]) => {
+    };
+    // FIXME: This error gets thrown on first launch / wallet init -- probably
```

Comment:
@dittonjs - please take a look at this when you get a chance (this is a best-guess temporary fix).